databricks/koalas/spark/accessors.py

```diff
@@ -851,6 +851,50 @@ def apply(self, func, index_col: Optional[Union[str, List[str]]] = None) -> "ks.
             )
         return output.to_koalas(index_col)
 
+    def repartition(self, num_partitions: int) -> "ks.DataFrame":
+        """
+        Returns a new DataFrame partitioned by the given partitioning expressions. The
+        resulting DataFrame is hash partitioned.
+
+        Parameters
+        ----------
+        num_partitions : int
+            The target number of partitions.
+
+        Returns
+        -------
+        DataFrame
+
+        Examples
+        --------
+        >>> kdf = ks.DataFrame({"age": [5, 5, 2, 2],
+        ...         "name": ["Bob", "Bob", "Alice", "Alice"]}).set_index("age")
+        >>> kdf  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
Shall we use `.sort_index()` here as well so that we can see the same as the repartitioned result?

Reply:
Makes sense! Updated.